en/cloud_development_setup/instructions.md

```diff
@@ -29,7 +29,7 @@ All you need to do is find a directory in which you want to create the `virtuale
 For this tutorial we will be using a new directory `djangogirls` from your home directory:
 
 {% filename %}command-line{% endfilename %}
-```
+```shell
```

Comment:
This is very common in this file, so I'm not commenting it on every line.
````suggestion
```console
````